crates/typst/src/export/pdf/mod.rs

```diff
@@ -217,7 +233,9 @@ fn write_page_labels(ctx: &mut PdfContext) -> Vec<(NonZeroUsize, Ref)> {
 
     for (i, page) in ctx.pages.iter().enumerate() {
         let nr = NonZeroUsize::new(1 + i).unwrap();
-        let Some(label) = &page.label else { continue };
+        let Some(label) = &page.label else {
+            continue;
+        };
```

Comment:
This should be changed back.

Reply:
`cargo fmt --check --all` complains about this on my end. I've reverted it anyway, but just so you know.

Reply:
Yep, CI failed because of that

Reply:
Is it maybe because you still have the semicolon behind the continue? That will force a line break in rustfmt. If it's completely reverted it should be fine since CI passes on main.

Reply:
ah yeah that was it